cpp/src/parquet/CMakeLists.txt

```diff
@@ -335,7 +336,7 @@ install(FILES "${CMAKE_CURRENT_BINARY_DIR}/parquet_version.h"
 add_parquet_test(internals-test
                  SOURCES
                  bloom_filter_test.cc
-                 bloom_filter_reader_test.cc
+                 bloom_filter_parquet_test.cc
```

Comment:
What above merging `bloom_filter_parquet_test.cc` into `bloom_filter_test.cc`?

Reply:
Actually `bloom_filter_test.cc` is "BlockSplitBloomFilter Test", I think it would be confusion to move into it.

Reply:
Can we remove "parquet" in the test name? It's superfluous.

Reply:
How about:

```
bloom_filter_test.cc -> block_split_filter_test.cc
bloom_filter_reader_test.cc -> bloom_filter_test.cc
```